### ISSUE TITLE
A loss of visual focus occurs on “Help-info” button in Signposts tab

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1932]
URL: http://clarity-ks.surge.sh/signposts

Reproduced Steps:
1. Login Clarity Kitchen Sink with web browser.
2. Go to Signposts tab.
3. Use the tab key to access all the interactive elements in Signposts tab.
4. Check visual indication of the current focus.

Actual Results:
A loss of visual focus occurs on “help-info” button. 
For more details, please refer to “Signposts-LossFocus.png”.

Expected Results:
There should have visual indication of the current focus.

Violation of Section 508 Standards: 
1194.21(c) A well-defined on-screen indication of the current focus shall be provided that moves among interactive interface elements as the input focus changes. The focus shall be programmatically exposed so that assistive technology can track focus and focus changes. 

User Impact: Blind.

![image.png](https://images.zenhubusercontent.com/582f4cffb9dc3c6220222f57/fc32fc3c-70be-426e-b139-40d2779fc03d)

**Comment by mathisscott** [comment]
duplicate #1274
